prometheus/src/main/java/filodb/prometheus/antlr/PromQL.g4

```diff
@@ -116,7 +116,7 @@ literal: NUMBER | STRING;
 
 // Number format is non-standard.
 // It doesn't support hex, NaN, Inf, and it doesn't require a digit after the decimal point.
-NUMBER: [-+] ? (
+NUMBER: (
```

Comment:
Why was this removed? We need to support numbers like `+1` and `-5`

Reply:
Maybe good to add a test case for it.

Reply:
Expressions like `+1`  get parsed to to unaryOperation using `unaryOp vectorExpression`. I have added test cases for this. I removed it because parsing of expressions like '1+1' was failing as the operator was getting matched to NUMBER because of [-+]